Source/monster.cpp

```diff
@@ -3726,13 +3726,6 @@ void InitMonsterGFX(int monst)
 		LevelMonsterTypes[monst].Anims[anim].Rate = MonstersData[mtype].Rate[anim];
 	}
 
-	LevelMonsterTypes[monst].mMinHP = MonstersData[mtype].mMinHP;

```

Comment:
why is 3729, 3730 and 3735 being deleted?

Reply:
Because all it really does is precalculate Diablo's HP while initializing monster graphics, which represents an unnecessary level of misdirection that enabled this error to exist in the first place.